docs/README.md

```diff
@@ -219,23 +219,25 @@ You can define rules, using regex (regular expressions), to control what log dat
 
 For example, you can identify certain types of logs that are noisy and not needed at all, and then write a regex pattern to match those files and preclude them from collection. Conversely, you can use regex expressions to match the log lines that you do want to include, and collect **only** those log lines.
 
-Additionally, you can use the environment variable LOGDNA_REDACT_REGEX to remove certain parts of a log line. Any redacted data is replaced with [REDACTED]. Redacting information is recommended when logs might contain PII (Personally Identifiable Information).
+Additionally, you can use the environment variable `LOGDNA_REDACT_REGEX` to remove certain parts of a log line. Any redacted data is replaced with [REDACTED]. Redacting information is recommended when logs might contain PII (Personally Identifiable Information).
 
 In general, using Agent-level environment variables to define what log data does not ever leave the data center or environment allows a higher level of control over sensitive data, and can also reduce any potential data egress fees.
 
 You can use regex pattern matching via environment variables to:
- * include *only* specific log lines (with `LOGDNA_LINE_INCLUSION_REGEX`)
- * exclude specific log lines (with `LOGDNA_LINE_EXCLUSION_REGEX`)
- * redact parts of a log line (with `LOGDNA_REDACT_REGEX`)
- * preclude entire log files from being monitored `(LOGDNA_EXCLUSION_REGEX_RULES)`
 
-To access our library of common regex patterns, refer to [our regex library documentation](REGEX.md).
+* include *only* specific log lines (with `LOGDNA_LINE_INCLUSION_REGEX`)
+* exclude specific log lines (with `LOGDNA_LINE_EXCLUSION_REGEX`)
+* redact parts of a log line (with `LOGDNA_REDACT_REGEX`)
+* preclude entire log files from being monitored `(LOGDNA_EXCLUSION_REGEX_RULES)`
```

Comment:
I would avoid mentioning here `LOGDNA_EXCLUSION_REGEX_RULES` (and `LOGDNA_INCLUSION_REGEX_RULES`) that targets file names. Include and exclude lists for directories and files are usually more comfortable to be specified using glob patterns.

I think it's better if we dedicate this section entirely to exclusion/inclusion/redaction of log lines instead.

Reply:
OK, sounds reasonable.

Reply:
I'll remove it 👍